stories/Tooltip/Tooltip.stories.js

```diff
@@ -12,6 +12,12 @@ import EsempiInterattivi from './docs/EsempiInterattivi.md'
 const EsempiComponent = () => {
   const ids = ['Example1', 'Example2', 'Example3']
 
+  ids.map((id, i) => {
```

Comment:
Not sure of this strategy.
These ids are associated with several anchor tags in the component: perhaps refs are missing? Can't we use refs instead?